scripts/download_data.py

```diff
@@ -231,6 +258,31 @@ def downloadBams(self):
             pysam.index(fileName.encode('utf-8'))
         escapeDir()
 
+    def downloadFastas(self):
+        dirList = [
+            self.args.dir_name, 'references', 'ebi']
+        mkdirAndChdirList(dirList)
+        cleanDir()
+        baseUrl = 'http://www.ebi.ac.uk/ena/data/view/'
+        for chromosome in self.chromosomes:
+            accession = self.accessions[chromosome]
+            path = os.path.join(baseUrl, accession)
+            minPos = self.chromMinMax.getMinPos(chromosome)
+            maxPos = self.chromMinMax.getMaxPos(chromosome)
+            args = urllib.urlencode({
+                'display': 'fasta',
+                'range': '{}-{}'.format(minPos, maxPos)})
```

Comment:
I think this should be ``(0, maxPos)``; otherwise, we'll hit some nasty offsetting problems.